server/models/ingredient.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const IngredientSchema = new mongoose.Schema({
+    valid: {
```

Comment:
Use this convention whenever there's a boolean
```suggestion
    isValid: {
```